controllers/disruption_controller.go

```diff
@@ -94,7 +94,7 @@ func (r *DisruptionReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	r.log = r.BaseLog.With("disruptionName", req.Name, "disruptionNamespace", req.Namespace)
 
 	// reconcile metrics
-	r.handleMetricSinkError(r.MetricsSink.MetricReconcile())
+	r.handleMetricSinkError(r.MetricsSink.MetricReconcile([]string{"controller", r.MetricsSink.GetSinkName()}))
```

Comment:
I don't think controller is the right tag to send here? Maybe I'm missing something?
Same comment for the disruption cron controller on the same call.